src/candela/VisComponent/mixin/AutoResize.js

```diff
@@ -0,0 +1,16 @@
+import Resize from './Resize';
+
+let AutoResize = Base => class extends Resize(Base) {
```

Comment:
I agree. I think I may have left it out because I've observed that the ``Resize`` mixin causes a resize event to be fired as soon as the element is initialized, which for ``AutoResize`` components will sort of repeat that initial work. But (1) I am not certain that it is guaranteed that a resize event will fire and (2) the tiny bit of extra initialization overhead is probably worth making it explicit that ``AutoResize`` works off the conceptual base provided by ``InitSize``.